sklearn/notebooks/Online Prediction with scikit learn.ipynb

```diff
@@ -0,0 +1,698 @@
+      "      \"deploymentUri\": \"gs://true-ability-192918/\",\r\n",
+      "      \"createTime\": \"2018-01-31T23:08:22Z\",\r\n",
+      "      \"etag\": \"tc3SVp7707A=\",\r\n",
+      "      \"framework\": \"SCIKIT_LEARN\"\r\n",
+      "    }\r\n",
+      "  }\r\n",
+      "}\r\n"
+     ]
+    }
+   ],
+   "source": [
+    "! curl -X POST -H \"Content-Type: application/json\" \\\n",
+    "   -d '{\"name\": \"'$VERSION_NAME'\", \"deploymentUri\": \"gs://'$BUCKET_ID'/\", \"runtimeVersion\": \"1.4\", \"framework\": \"SCIKIT_LEARN\"}' \\\n",
+    "   -H \"Authorization: Bearer `gcloud auth print-access-token`\" \\\n",
+    "    https://ml.googleapis.com/v1/projects/$PROJECT_ID/models/$MODEL_NAME/versions"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Part 5: Make an online prediction\n",
+    "It’s time to make an online prediction with your newly deployed model. Before you begin, you'll need to take some of the test data and prepare it, so that the test data can be used by the deployed model.\n",
+    "\n",
+    "To get online predictions, the data needs to be converted from a numpy array to a json array. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Show a person that makes <=50K:\n",
+      "\tFeatures: [25, 4, 226802, 1, 7, 4, 7, 3, 2, 1, 0, 0, 40, 38] --> Label: False\n",
+      "\n",
+      "Show a person that makes >50K:\n",
+      "\tFeatures: [28, 2, 336951, 7, 12, 2, 11, 0, 4, 1, 0, 0, 40, 38] --> Label: True\n"
+     ]
+    }
+   ],
+   "source": [
+    "import json\n",
+    "\n",
+    "data = []\n",
+    "for i in range(len(test_features)):\n",
+    "  data.append([])\n",
+    "  for col in COLUMNS[:-1]: # Ignore the 'income-level' column as it is not in the feature set.\n",
+    "    # Convert from numpy integers to standard integers\n",
+    "    data[i].append(int(np.uint64(test_features[col][i]).item()))\n",
+    "\n",
+    "# Write the test data to a json file\n",
+    "with open('data.json', 'w') as outfile:\n",
+    "  json.dump(data, outfile)\n",
+    "\n",
+    "# Get one person that makes <=50K and one that makes >50K to test our model.\n",
+    "print('Show a person that makes <=50K:')\n",
+    "print('\\tFeatures: {0} --> Label: {1}\\n'.format(data[0], test_labels[0]))\n",
+    "\n",
+    "with open('less_than_50K.json', 'w') as outfile:\n",
+    "  json.dump(data[0], outfile)\n",
+    "\n",
+    "  \n",
+    "print('Show a person that makes >50K:')\n",
+    "print('\\tFeatures: {0} --> Label: {1}'.format(data[2], test_labels[2]))\n",
+    "\n",
+    "with open('more_than_50K.json', 'w') as outfile:\n",
+    "  json.dump(data[2], outfile)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Use Gcloud to make online predictions\n",
```

Comment:
It's pretty weird that you are capitalizing the g in `gcloud` here.

Reply:
Fixed